awx/main/models/credential/__init__.py

```diff
@@ -47,12 +47,12 @@
 )
 from awx.main.models import Team, Organization
 from awx.main.utils import encrypt_field
-from . import injectors as builtin_injectors
+from awx_plugins.credentials import injectors as builtin_injectors
 
 __all__ = ['Credential', 'CredentialType', 'CredentialInputSource', 'build_safe_env']
 
 logger = logging.getLogger('awx.main.models.credential')
-credential_plugins = dict((ep.name, ep.load()) for ep in iter_entry_points('awx.credential_plugins'))
+credential_plugins = dict((ep.name, ep.load()) for ep in iter_entry_points('awx_plugins.credentials'))
```

Comment:
FTR, this will conflict with #15441.

Reply:
I would like to get #15441 merged, but first things first. Let's get a stable foundation of awx and awx-plugins and then make these sort of changes.